client/containers/DashboardActivity/DashboardActivity.tsx

```diff
@@ -43,7 +46,30 @@ const DashboardActivity = (props: Props) => {
 	});
 
 	return (
-		<DashboardFrame className="dashboard-activity-container" title="Activity">
+		<DashboardFrame
+			className="dashboard-activity-container"
+			title="Activity"
+			controls={
+				canEnrollInActivityDigestEmails && (
```

Comment:
❗️ We also only want to show this the Community scope, right? As-is this appears on Collection/Pub activity dashboards but still targets the parent Community.